components/feature/app-links/src/test/java/mozilla/components/feature/app/links/AppLinkRedirectTest.kt

```diff
@@ -74,5 +74,6 @@ class AppLinkRedirectTest {
         var appLink = AppLinkRedirect(appIntent = null, fallbackUrl = null, marketplaceIntent = mock())
         assertTrue(appLink.hasMarketplaceIntent())
         assertTrue(appLink.isRedirect())
+        assertTrue(appLink.hasMarketplaceIntent())
     }
 }
```

Comment:
nit: `\n`? Not sure how lint check didn't call this out :)

Reply:
Thanks for catching this.  I'll add the `\n`